From 7cf5a3612354fa7dc760cd905e27658c4c8d68cd Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Wed, 12 Apr 2017 17:05:47 +0300
Subject: [PATCH 1664/5242] MLK-14586: ARM: imx: Fix path to serial port for
 pm save/restore

commit  fbed909966dfedaeef7051d94745b737990055f8 from
https://source.codeaurora.org/external/imx/linux-imx.git

The imx7 sleep code will save and restore UART1 registers on
suspend/resume. It does this by fetching the address base using an
absolute devicetree path. Fix that path in 4.9 where the DTS is closer
to the one in upstream.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/pm-imx7.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/pm-imx7.c b/arch/arm/mach-imx/pm-imx7.c
index 06ee8c5..7fde0b6 100644
--- a/arch/arm/mach-imx/pm-imx7.c
+++ b/arch/arm/mach-imx/pm-imx7.c
@@ -1165,7 +1165,7 @@ void __init imx7d_pm_init(void)
 	WARN_ON(!ocram_saved_in_ddr);
 
 	np = of_find_node_by_path(
-		"/soc/aips-bus@30800000/spba-bus@30800000/serial@30860000");
+		"/soc/aips-bus@30800000/serial@30860000");
 	if (np)
 		console_base = of_iomap(np, 0);
 
-- 
1.7.9.5

